diff --git a/node_modules/@react-navigation/elements/lib/commonjs/Background.js b/node_modules/@react-navigation/elements/lib/commonjs/Background.js
index 2d86363..0b4156d 100644
--- a/node_modules/@react-navigation/elements/lib/commonjs/Background.js
+++ b/node_modules/@react-navigation/elements/lib/commonjs/Background.js
@@ -15,13 +15,12 @@ function Background(_ref) {
     style,
     ...rest
   } = _ref;
-  const {
-    colors
-  } = (0, _native.useTheme)();
+  const theme = (0, _native.useTheme)();
+  const backgroundColor = theme?.colors?.background ?? '#f2f2f2';
   return /*#__PURE__*/React.createElement(_reactNative.View, _extends({}, rest, {
     style: [{
       flex: 1,
-      backgroundColor: colors.background
+      backgroundColor
     }, style]
   }));
 }
diff --git a/node_modules/@react-navigation/elements/lib/module/Background.js b/node_modules/@react-navigation/elements/lib/module/Background.js
index aaa10a2..7b53c58 100644
--- a/node_modules/@react-navigation/elements/lib/module/Background.js
+++ b/node_modules/@react-navigation/elements/lib/module/Background.js
@@ -7,13 +7,12 @@ export default function Background(_ref) {
     style,
     ...rest
   } = _ref;
-  const {
-    colors
-  } = useTheme();
+  const theme = useTheme();
+  const backgroundColor = theme?.colors?.background ?? '#f2f2f2';
   return /*#__PURE__*/React.createElement(View, _extends({}, rest, {
     style: [{
       flex: 1,
-      backgroundColor: colors.background
+      backgroundColor
     }, style]
   }));
 }
diff --git a/node_modules/@react-navigation/elements/src/Background.tsx b/node_modules/@react-navigation/elements/src/Background.tsx
index 986609c..04e67b2 100644
--- a/node_modules/@react-navigation/elements/src/Background.tsx
+++ b/node_modules/@react-navigation/elements/src/Background.tsx
@@ -7,12 +7,13 @@ type Props = ViewProps & {
 };
 
 export default function Background({ style, ...rest }: Props) {
-  const { colors } = useTheme();
+  const theme = useTheme();
+  const backgroundColor = theme?.colors?.background ?? '#f2f2f2';
 
   return (
     <View
       {...rest}
-      style={[{ flex: 1, backgroundColor: colors.background }, style]}
+      style={[{ flex: 1, backgroundColor }, style]}
     />
   );
 }
diff --git a/node_modules/@react-navigation/elements/src/Header/HeaderBackButton.tsx b/node_modules/@react-navigation/elements/src/Header/HeaderBackButton.tsx
index 7c2a541..75fe28a 100644
--- a/node_modules/@react-navigation/elements/src/Header/HeaderBackButton.tsx
+++ b/node_modules/@react-navigation/elements/src/Header/HeaderBackButton.tsx
@@ -1,4 +1,4 @@
-import { useTheme } from '@react-navigation/native';
+import { useTheme, DefaultTheme } from '@react-navigation/native';
 import * as React from 'react';
 import {
   Animated,
@@ -33,7 +33,8 @@ export default function HeaderBackButton({
   testID,
   style,
 }: HeaderBackButtonProps) {
-  const { colors } = useTheme();
+  const theme = useTheme();
+  const colors = theme.colors ?? DefaultTheme.colors;
 
   const [initialLabelWidth, setInitialLabelWidth] = React.useState<
     undefined | number
diff --git a/node_modules/@react-navigation/elements/src/Header/HeaderBackground.tsx b/node_modules/@react-navigation/elements/src/Header/HeaderBackground.tsx
index 0f8696b..04db21a 100644
--- a/node_modules/@react-navigation/elements/src/Header/HeaderBackground.tsx
+++ b/node_modules/@react-navigation/elements/src/Header/HeaderBackground.tsx
@@ -15,16 +15,17 @@ type Props = Omit<ViewProps, 'style'> & {
 };
 
 export default function HeaderBackground({ style, ...rest }: Props) {
-  const { colors } = useTheme();
+  const theme = useTheme();
+  const colors = theme?.colors;
 
   return (
     <Animated.View
       style={[
         styles.container,
         {
-          backgroundColor: colors.card,
-          borderBottomColor: colors.border,
-          shadowColor: colors.border,
+          backgroundColor: colors?.card ?? '#ffffff',
+          borderBottomColor: colors?.border ?? '#d8d8d8',
+          shadowColor: colors?.border ?? '#d8d8d8',
         },
         style,
       ]}
diff --git a/node_modules/@react-navigation/elements/src/Header/HeaderTitle.tsx b/node_modules/@react-navigation/elements/src/Header/HeaderTitle.tsx
index 7dda981..c40ab1b 100644
--- a/node_modules/@react-navigation/elements/src/Header/HeaderTitle.tsx
+++ b/node_modules/@react-navigation/elements/src/Header/HeaderTitle.tsx
@@ -1,4 +1,4 @@
-import { useTheme } from '@react-navigation/native';
+import { useTheme, DefaultTheme } from '@react-navigation/native';
 import * as React from 'react';
 import {
   Animated,
@@ -15,7 +15,8 @@ type Props = Omit<TextProps, 'style'> & {
 };
 
 export default function HeaderTitle({ tintColor, style, ...rest }: Props) {
-  const { colors } = useTheme();
+  const theme = useTheme();
+  const colors = theme.colors ?? DefaultTheme.colors;
 
   return (
     <Animated.Text
